src/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -348,6 +349,27 @@ private static void loadSwitchAndItemUseMaterialsLists() {
 		}
 	}
 
+	public static EnumSet<EntityType> toEntityTypeEnumSet(List<String> entityList) {
+		EnumSet<EntityType> entities = EnumSet.noneOf(EntityType.class);
+		for (String entityName : entityList) {
+			EntityType type = EntityType.valueOf(entityName);
```

Comment:
The returned type here isn't nullable, instead it needs to catch the IllegalArgumentException

Reply:
Good catch.